binary-chop/Makefile

```diff
@@ -4,7 +4,7 @@
 
 # We enable execution tree (tree.dot) output as the examples
 # in this directory are small.
-EXTRA_OPTIONS=-output-tree -interpolation-stat -write-pcs -use-query-log=all:pc,all:smt2 -special-function-bound-interpolation  -no-bound-check #-debug-subsumption=1 #-exit-on-error  
+EXTRA_OPTIONS=-output-tree -interpolation-stat -write-pcs -use-query-log=all:pc,all:smt2 -special-function-bound-interpolation  -no-bound-check -debug-subsumption=1 #-exit-on-error  
```

Comment:
@rasoolmaghareh It is fine this time, but this modification should actually be in a separate PR,